blocks/footer/footer.css

```diff
@@ -18,3 +18,57 @@ footer .footer p {
     padding: 40px 32px 24px;
   }
 }
+
+footer .icon-footer-tia img {
+    width: 135px;
+    height: 35px;
+    margin-bottom: 3rem !important;
```

Comment:
1. Can we bypass the `!important` with higher specificity?
2. Not sure it is a good idea to use `rem` for margin/padding as it will rely on user's set font-size. If they set a higher font size in the browser, the spacing will break in the app.

Reply:
@mbradoschi i reused what was defined in their CSS, if you have other suggestion please advise which to use.